Card/src/test/java/com/paypal/android/card/CardClientUnitTest.kt

```diff
@@ -21,15 +21,15 @@ class CardClientUnitTest {
     private val orderID = "sample-order-id"
 
     private val cardAPI = mockk<CardAPI>(relaxed = true)
-    private val cardResult = CardResult.Success("orderId", OrderStatus.APPROVED)
+    private val cardResult = CardResult("orderId", OrderStatus.APPROVED)
 
     private val testCoroutineDispatcher = TestCoroutineDispatcher()
```

Comment:
```suggestion
```
Don't think we need this anymore

Reply:
Yeah we can remove since we're mocking this co-routine. This can sometimes be useful but I forget why.